frontend/src/pages/projects/screens/detail/storage/data.ts

```diff
@@ -15,16 +15,23 @@ export const columns: SortableData<PersistentVolumeClaimKind>[] = [
     sortable: (a, b) =>
       getDisplayNameFromK8sResource(a).localeCompare(getDisplayNameFromK8sResource(b)),
   },
+  {
+    field: 'storage',
+    label: 'Storage class',
+    width: 30,
+    sortable: (a, b) =>
+      (a.spec.storageClassName ?? '').localeCompare(b.spec.storageClassName ?? ''),
```

Comment:
this will not always be accurate. sort via the config display name stored in the annotation. Fall back to the storage class k8s name

Reply:
I checked this ,  The `a` here is of the type `PersistentVolumeClaimKind` there is no config display name stored in the annotation 
I think you are mentioning about the type `StorageClassKind[]`???
We only have `name` which is stored in `a.spec.storageClassName`

Reply:
right but you are sorting based on the name in the config, so you may need to get it in their to be able to sort it. for example creating a type for the table like

```ts
type StorageTableData = {
   pvc: PersistentVolumeClaimKind;
   storageClass: StorageClassKind;
}